engine-standalone-tracing/src/types.rs

```diff
@@ -231,19 +252,19 @@ impl TransactionTrace {
     }
 
     /// Returns the return as a str reference.
```

Comment:
Update the doc comment

Reply:
Done in https://github.com/aurora-is-near/aurora-engine/pull/384/commits/3bd3c9f1f2b34c5a12759c99f9c9235026c410d5